roles/eda/defaults/main.yml

```diff
@@ -12,9 +12,10 @@ image_pull_secrets: []
 
 _image: quay.io/ansible/eda-server
 _image_version: main
+_image_version: "{{ lookup('env', 'DEFAULT_EDA_VERSION') or 'main' }}"
 
 _image_web: quay.io/ansible/eda-ui
-_image_web_version: main
+_image_web_version: "{{ lookup('env', 'DEFAULT_EDA_UI_VERSION') or 'main' }}"
```

Comment:
the tags in the builds as part of release use the `latest` tag

Reply:
See above [comment](https://github.com/ansible/eda-server-operator/pull/159#discussion_r1447492311).